generate/sam-2016-10-31.json

```diff
@@ -391,6 +457,63 @@
                     "UpdateType": "Immutable"
                 }
             }
+        },
+        "AWS::Serverless::Api.S3Location": {
```

Comment:
This is intentionally the same structure as Serverless::Function CodeUri property. We can merge them if needed.